src/main/reactjs/src/enums/app.ts

```diff
@@ -33,18 +33,18 @@ export enum PublicUIViews {
   ContactRequest = 'ContactRequest',
 }
 
-export enum TextBoxTypes {
+export enum CustomTextFieldTypes {
```

Comment:
Olisiko tässä kuitenkin TextFieldTypes parempi? Geneerisissä asioissa voisi käyttää varmaan samaa.

Reply:
Joo, ehkäpä. Mites tuo `CustomTextFieldErrors` hieman alempana? Näitä kuitenkin käytetään samassa paikassa, mikä puoltaisi samankaltaista nimeämistä - toisaalta tuo virheidenkäsittely on täysin meidän omaa logiikkaa.

Reply:
Joo,  Jos on meidän omaa logiikkaa, niin lisätään custom eteen, mutta jos on  ns. geneerinen/yleinen asia niin silloin ei laiteta customia ollenkaan. Miltäs kuulostaa?

Reply:
Mennään sillä ainakin nyt.